src/Presentation/Memo4me.UI/Components/Pages/Login.razor

```diff
@@ -0,0 +1,19 @@
+﻿@page "/login"
+@using Microsoft.AspNetCore.Authentication
+@rendermode InteractiveServer
+@inject NavigationManager UriHelper
+@inject IHttpContextAccessor HttpContextAccessor
+
+<h3>Login</h3>
+
+@code {
+    protected override async Task OnInitializedAsync()
```

Comment:
Could you please explain to me, when and in which conditions this method is invoked

Reply:
OnInitializedAsync() is a lifecycle method of the blazor component that is called once when the component is initialized and after it has been added to rendering
In my case i use this method to initialize the authentication process, sending the user to the login page